Store/StoreAnalyticsOrderTracker.php

```diff
@@ -48,9 +49,30 @@ public function getCommands()
 	}
 
 	// }}}
-	// {{{ protected function getOrderCommand()
+	// {{{ public function getFacebookPixelCommands()
 
-	protected function getOrderCommand()
+	public function getFacebookPixelCommands()
+	{
+		$commands = array($this->getGoogleAnalyticsOrderCommand());
+		foreach ($this->order->items as $item) {
+			$commands[] = $this->getGoogleAnalyticsOrderItemCommand($item);
```

Comment:
Is it correct to add GA commands to the facebook pixel API? If so, a comment is required.